addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js

```diff
@@ -9,9 +9,11 @@ const patchHrPresenceStatus = () => ({
     get color() {
         if (this.location) {
             let color = "text-muted";
-            if (this.props.record.data.hr_presence_state !== "to_define") {
+            console.log(this.props.record.data)
+            if (this.props.record.data.hr_presence_state !== "to_define" && this.props.record.data.hr_presence_state !== "offline") {
                 color = this.props.record.data.hr_presence_state === "present" ?  "text-success" : "text-warning";
             }
+            console.log(color)
```

Comment:
You forgot to remove some logs it seems :)
```suggestion
            if (this.props.record.data.hr_presence_state !== "to_define" && this.props.record.data.hr_presence_state !== "offline") {
                color = this.props.record.data.hr_presence_state === "present" ?  "text-success" : "text-warning";
            }
```